backend/grant/app.py

```diff
@@ -102,6 +102,9 @@ def register_blueprints(app):
     app.register_blueprint(blockchain.views.blueprint)
     app.register_blueprint(task.views.blueprint)
     app.register_blueprint(rfp.views.blueprint)
+    if E2E_TESTING:
```

Comment:
I would modify this to:

```if E2E_TESTING and DEBUG``` 

just to be extra paranoid.

Reply:
I'd also disable Sentry during E2E, just to spare some annoyance